test/monitoring/components_metrics_test.go

```diff
@@ -60,11 +64,30 @@ func testControlPlanePrometheusMetrics(t *testing.T, v1api v1.API) {
 			componentName: "calico-felix",
 			query:         "felix_active_local_endpoints",
 		},
+		{
+			componentName: "metallb",
+			query:         "metallb_bgp_session_up",
+			platforms:     "packet",
+		},
+		{
+			componentName: "contour",
+			query:         "contour_dagrebuild_timestamp",
+			platforms:     "packet,aws",
```

Comment:
Why not a slice or map?

Reply:
This way it is easier for the user writing test.

Reply:
We could use strings for all values when programming, but it's generally not a good idea :smile: